suite/tests/client-interface/drx-scattergather.c

```diff
@@ -235,6 +241,20 @@ signal_handler_check_k1(int sig, siginfo_t *siginfo, ucontext_t *ucxt)
 #        endif
     SIGLONGJMP(mark, 1);
 }
+
+static void
+signal_handler_check_ymm1(int sig, siginfo_t *siginfo, ucontext_t *ucxt)
+{
+    kernel_fpstate_t *fp = (kernel_fpstate_t *)ucxt->uc_mcontext.fpregs;
+#        ifdef X64
+    if (((fp->xmm_space[8] >> 31) & 0x1) != 0) {
+#        else
+    if (((fp->_xmm[2].element[3] >> 31) & 0x1) != 0) {
```

Comment:
Please add either a comment pointing out where the code is that writes to this particular precise register part (and probably a comment by that code too about this linked code here), or use named constants if that's straightforward.

Ditto for the xmm_space[8]'s 32nd bit above.

Reply:
Done.